.clang-format

```diff
@@ -0,0 +1,6 @@
+BasedOnStyle: Google
```

Comment:
It's necessary to fix the sorting order of the includes.
I suggest:

1. `".*"`                            // relative includes
2. `<yaclib\/.*>`               // our library public includes
3. `<[[:alpha:]_\/]+>`        // STL
4. `<.*>`                          // other libraries in our case: system headers